website/docs/docs/on-premises/installation/migrating-from-pact-broker.md

```diff
@@ -0,0 +1,34 @@
+---
+id: migrating
+title: Migrating from Pact Broker
+---
+
+# Migrating from Pact Broker
+
+It is possible to migrate from using the open source Pact Broker to Pactflow, keeping all existing data and making it available in Pactflow. The migration process uses the existing Pact Broker database, database migrations are then performed to add additional tables and data used for Pactflow features.
```

Comment:
Assume Postgres db’s only for migration.

should we state that, as I believe support has dropped for MySQL?

https://github.com/pact-foundation/pact_broker#rolling-your-own

Reply:
Ah yes, this is a good call out. We'll want a DB dump from Postgres.

Perhaps we just add one of those note sections e.g.

```
:::note
Migration is only supported for a Pact Broker running a Postgres database.
:::
```

Reply:
Good idea, yeah I interpret the same thing based on those docs.

Reply:
@mefellows would there be any other dbs supported? Or just Postgres? I know that when I run pactflow locally it's set up as an sqlite db🤔. wondering if people would use that or anything else we could support when using Pact Broker?

Reply:
I don't think we could reliably import an sqlite DB. We certainly don't recommend running it in "real life" (although people do!).

Reply:
Agreed.